.github/workflows/sdk.yaml

```diff
@@ -18,4 +18,28 @@ jobs:
       - name: Run npm Install
         run: npm install
 
-    # TODO: Check Lint
+      - name: Unit Tests
+        run: npm run test
+
+  # Checks code formatting, fails if there are changes after applying prettier.
+  # Based on this example here:
+  # https://github.com/creyD/prettier_action?tab=readme-ov-file#example-4-dry-run
+  prettier:
```

Comment:
Same job that we have on the raygun4node project, checks prettier format on each push/pr